frontend/src/toolbar/elements/ScrollDepth.tsx

```diff
@@ -0,0 +1,121 @@
+import { useValues } from 'kea'
+import { useEffect, useState } from 'react'
+
+import { heatmapLogic } from '~/toolbar/elements/heatmapLogic'
+
+import { toolbarConfigLogic } from '../toolbarConfigLogic'
+
+function ScrollDepthMouseInfo(): JSX.Element | null {
+    const { posthog } = useValues(toolbarConfigLogic)
+    const { heatmapElements, rawHeatmapLoading } = useValues(heatmapLogic)
+
+    // Track the mouse position and render an indicator about how many people have scrolled to this point
+    const [mouseY, setMouseY] = useState<null | number>(0)
+
+    useEffect(() => {
+        const onMove = (e: MouseEvent): void => {
+            setMouseY(e.clientY)
+        }
+
+        window.addEventListener('mousemove', onMove)
+        return () => {
+            window.removeEventListener('mousemove', onMove)
+        }
+    }, [])
+
+    if (!mouseY) {
+        return null
+    }
+
+    const scrollOffset = (posthog as any).scrollManager.scrollY()
+    const scrolledMouseY = mouseY + scrollOffset
+
+    const elementInMouseY = heatmapElements.find((x, i) => {
+        const lastY = heatmapElements[i - 1]?.y ?? 0
+        return scrolledMouseY >= lastY && scrolledMouseY < x.y
+    })
+
+    const maxCount = heatmapElements[0]?.count ?? 0
+    const percentage = ((elementInMouseY?.count ?? 0) / maxCount) * 100
```

Comment:
not blocking but this could mostly all be in the logic?

Reply:
My comments on your PR coming to haunt me. Good call 😅

Reply:
We've all got to ask WWBD